opus/import/table_schemas/obs_pds.json

```diff
@@ -177,7 +177,7 @@
         "pi_display": 1,
         "pi_display_results": 1,
         "pi_form_type": "STRING",
-        "pi_intro": null,
+        "pi_intro": "The Primary File SpecX has different filename extensions for each mission and instrument and in many cases specifies the label file instead of the data file. Thus we suggest that you eliminate the extension from your search and use the \"contains\" query type to guarantee the best match.",
```

Comment:
SpecX ==> Spec
has different ==> uses different
each mission and instrument ==> different data product types,
eliminate ==> do not include
from ==> in
and use ==> and that you use

Reply:
Made all changes except different data product types, which is now "different filename extensions depending on the mission and instrument, and in many cases"